### PR TITLE
Create EternalCoderss.json

### PR DESCRIPTION
**Pull request by EternalCoderss**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3741"></a>

### CODE REVIEW

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
change the file name to `data/EternalCoderss.json` so that it's in the correct directory

File: EternalCoderss.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/shivam-choudhary1/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Hashnode Blog",
+      "url": "hhttps://hashnode.com/@shivi",
```
Comment (Aadarsh805) [inline EternalCoderss.json:44]: Typo in the Hashnode URL, change it to `https://hashnode.com/@shivi`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/EternalCoderss.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
     },
     {
       "name": "Hashnode Blog",
-      "url": "hhttps://hashnode.com/@shivi",
+      "url": "hhttps://hashnode.com/shivi",
```
Comment (Aadarsh805) [inline data/EternalCoderss.json:44]: there's still a extra `h` in the https

it should be `https://hashnode.com/@shivi`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/EternalCoderss.json (hunk 2 above)
Comment (Aadarsh805) [reply data/EternalCoderss.json:44]: keep the `@`

**Comment by EternalCoderss** [comment]
Everything is Good

**Pull request closed without merging**